README.md

```diff
@@ -0,0 +1,3 @@
+## Install
+
+After download. run `sh link_dotfiles.sh` in the dotfiles root directory.
```

Comment:
```suggestion
After download. run `sh link_dotfiles.sh` in the dotfiles root directory.
Be careful because it will overwrite existing dotfiles in the path ~ with the dotfiles in this repo.
```